shadows/framework/src/main/java/org/robolectric/shadows/ShadowParcel.java

```diff
@@ -23,13 +26,48 @@
 import org.robolectric.annotation.RealObject;
 import org.robolectric.util.ReflectionHelpers;
 
+/**
+ * Shadow for {@link android.os.Parcel}.
+ */
 @Implements(Parcel.class)
 @SuppressWarnings("unchecked")
 public class ShadowParcel {
   @RealObject private Parcel realObject;
   private static final Map<Long, ByteBuffer> NATIVE_PTR_TO_PARCEL = new LinkedHashMap<>();
   private static long nextNativePtr = 1; // this needs to start above 0, which is a magic number to Parcel
 
+  @Implementation(maxSdk = JELLY_BEAN_MR2)
+  public <T extends Parcelable> T readParcelable(ClassLoader loader) throws IllegalAccessException, NoSuchFieldException, ClassNotFoundException {
```

Comment:
This looks like it duplicates behavior from Jelly Bean's `readParcelable()`, no? Is it necessary, or is implementing `readParcelableCreator()` enough?

Reply:
I was wrong, it should be `JELLY_BEAN_MR1`.